tests/test-awesomerc.lua

```diff
@@ -42,22 +36,20 @@ local steps = {
         assert(old_c)
 
         -- Test layout
-        -- local cb = get_callback({modkey}, " ")
-        -- assert(cb)
 
         --TODO use the key once the bug is fixed
         local l = old_c.screen.selected_tag.layout
         assert(l)
 
-        -- cb()
+        --awful.key.execute({modkey}, " ")
         awful.layout.inc(1)
```

Comment:
Maybe you forget to

    git fetch origin
    git rebase -i origin/master

Yes, git can be a git ;)